src/common/ui/code.vue

```diff
@@ -90,12 +90,16 @@ function getHandler(key) {
   CodeMirror.commands[key] = getHandler(key);
 });
 Object.assign(CodeMirror.keyMap.sublime, {
-  Tab: 'indentMore',
   'Shift-Ctrl-/': 'commentSelection',
 });
 CodeMirror.commands.commentSelection = cm => {
   cm.blockComment(cm.getCursor('from'), cm.getCursor('to'), { fullLines: false });
 };
+// pressing Tab key inside a line with no selection will reuse indent type (tabs/spaces)
+const { insertTab, insertSoftTab } = CodeMirror.commands;
+CodeMirror.commands.insertTab = () => (
+  this.cm.options.indentWithTabs ? insertTab() : insertSoftTab()
+);
```

Comment:
`this` does not work here and commands must be called with `cm` as a parameter:

```js
CodeMirror.commands.insertTab = cm => (
  cm.options.indentWithTabs ? insertTab(cm) : insertSoftTab(cm)
)
```

Didn't dig into the code so I'm not sure if there are other potential problems here.

Reply:
It was a bad copypaste, sorry.